From 289123805bea83cc4c0a9a64722a22fcdf7af5b9 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Thu, 28 Dec 2017 21:14:54 -0800
Subject: [PATCH 3/3] buildit: Use -undefined dynamic_lookup when linking

This parallels the change made in 71ba2871943cec379da0585c16f69fb5ac5a884b

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
---
 lib/buildit | 1 +
 1 file changed, 1 insertion(+)

diff --git libcxxabi-6.0.0.src/lib/buildit libcxxabi-6.0.0.src/lib/buildit
index 508a805..d03e0e6 100755
--- libcxxabi-6.0.0.src/lib/buildit
+++ libcxxabi-6.0.0.src/lib/buildit
@@ -52,6 +52,7 @@ case $TRIPLE in
         -current_version ${RC_ProjectSourceVersion} \
         -compatibility_version 1 \
         -install_name /usr/lib/libc++abi.dylib \
+        -Wl,-undefined,dynamic_lookup \
         -lSystem"
 	if [ -f "${SDKROOT}/usr/local/lib/libCrashReporterClient.a" ]
 	then
-- 
2.15.1

